packages/core/src/app/ovm/integer-quantifiers.ts

```diff
@@ -0,0 +1,62 @@
+import { QuantifierResult, Quantifier } from '../../types'
+
+// Helper function which returns an array of numbers, starting at start, ending at end, incrementing by 1.
+// Eg. [0, 1, 2,...end]
+const range = (start: number, end: number) =>
+  Array(end - start)
+    .fill(start)
+    .map((x, y) => x + y)
+
+/*
+ * The parameter type for `getAllQuantified(...)` in the IntegerRangeQuantifier.
+ */
+interface IntegerRangeParameters {
+  start: number
+  end: number
+}
+
+/*
+ * The IntegerRangeQuantifier returns a range of integers between the start & end.
+ */
+export class IntegerRangeQuantifier implements Quantifier {
+  /**
+   * Returns a QuantifierResult where results are an array of integers from 0 to _lessThanThis. Eg. 0 to 3 = [0, 1, 2]
```

Comment:
I think the comments here are copy/pasted from `NonnegativeIntegerLessThanQuantifier`.